app.ts

```diff
@@ -0,0 +1,226 @@
+			for (const record of recordsvalue) {
+				DisplayContent += `<tr id="table-row-${record[0]}">`;
+				for (const column of record) {
+					DisplayContent += `<td align="center">${column}</td>`;
+				}
+				DisplayContent += '</tr>';
+				$("#wrapper-table-content-body").empty();
+				$("#wrapper-table-content-body").append(DisplayContent);
+				throw new Error("Error No data");
+			}
+			return [fromID, toID];
+		}
+
+		// Load Records Function
+		function RecordsFromCursor(cursor: number[]): Promise<number[]> {
+			cursor = cursor.sort((a, b) => { return a - b });
+			return LoadRecordsData(cursor[0], cursor[1]);
+			// throw new Error("Error");
+		}
+
+		// Handlers
+		async function LoadPageContent(fromID: number, toID: number): Promise<number[]> {
+			let DisplayContent = "";
+			const columns = await getColumnNamesCall();
+			for (const column of columns) {
+				DisplayContent += `<th align="center">${column}</th>`;
+				$("#wrapper-table-header-row").empty();
+				$("#wrapper-table-header-row").append(DisplayContent);
+			}
+			return LoadRecordsData(fromID, toID);
+		}
+
+		// Conversion Function
+		function ConvertNumber(input: string | number, parseAsInt: boolean = true): number {
+			switch (typeof input) {
+				case ('string'):
+					if (parseAsInt == true) {
+						return parseInt(input as string);
+					}
+					return parseFloat(input as string);
+				case ("number"):
+					return input as number;
+			}
+			return 0;
+		}
+
+		// Height Diplay Function
+		function calculateToId(fromId: number): number {
+			const possibleRecordsData = Math.max((window.innerHeight - ($("#form-content").innerHeight() as number)));
+			const possibleId = fromId + possibleRecordsData;
+			let recordDisplayset = 0;
+			switch (recordDisplayset) {
+				case 0:
+					window.innerHeight <= 646;
+					break;
+				case 1:
+					window.innerHeight <= 969;
+					break;
+				case 2:
+					window.innerHeight <= 1938;
+					break;
+				default:
+					recordDisplayset = 15;
+					break;
+			}
+			return recordDisplayset + possibleId;
+		}
+
+		//Next_Page_Resize Function
+		function nextPageResize(previous: number[]): number {
+			const fromID = ConvertNumber(previous.sort((a, b) => { return a - b })[0]);
+			const toID = ConvertNumber(previous.sort((a, b) => { return a - b })[1]);
+			const documentHeight = $(window).innerHeight() as number - ($(`#table-row-${fromID}`).height() as number);
+			for (let i = fromID; i <= toID; i++) {
+				const elementHeightOffset = ($(`#table-row-${i}`).offset() as JQueryCoordinates).top;
+				if (elementHeightOffset < documentHeight) continue;
+				return i;
+			}
+			return toID;
+		}
+
+		//Onload Function
+		window.onload = async () => {
+
+			//Previous_Page_Resize Function
+			function previousPageResize(previous: number[]): number[] {
+				const toId = calculateToId(previous[0] - (nextPageResize(previous) - previous[0]));
+				return [previous[0] - (nextPageResize(previous) - previous[0]), toId];
```

Comment:
You should rather store the result of `nextPageResize` and use that, than calling that function multiple times